checkov/common/graph/graph_builder/graph_components/blocks.py

```diff
@@ -41,7 +41,7 @@ def _extract_inner_attributes(self) -> Dict[str, Any]:
         attributes_to_add = {}
         for attribute_key in self.attributes:
             attribute_value = self.attributes[attribute_key]
-            if type(attribute_value) is list and len(attribute_value) > 0 and type(attribute_value[0]) is dict:
+            if isinstance(attribute_value, dict) or (isinstance(attribute_value, list) and len(attribute_value) > 0 and isinstance(attribute_value[0], dict)):
```

Comment:
`isinstance(attribute_value[0], dict))` wouldn't this break, if `attribute_value` is a dict?

Reply:
@gruebel It wouldn't break because it either a dict or it's a list, and if so - check `attribute_value[0]`

Reply:
Yeah, you are right. Didn't see the or at the beginning 😅